extras/paid-books.md

```diff
@@ -91,6 +91,11 @@ Name | Author | ISBN
 ### Cloud Computing
 
 ### Math (Linear Algebra)
+Name | Author | ISBN
+:-- | :--: | :--:
+[Linear Algebra Done Right third edition](http://www.amazon.in/Linear-Algebra-Right-Undergraduate-Mathematics/dp/0387982582) | Sheldon Axler | 0387982582
+
```

Comment:
nice work @arpit1997! I think if you remove this line, the table formatting should work on both books.